tests/pytest/core/test_views.py

```diff
@@ -24,6 +24,23 @@ def test_homepage_single_agency(mocker, client):
     assert response.url == agencies[0].index_url
 
 
+@pytest.mark.django_db
+def test_homepage_single_agency_single_verifier(mocker, client):
+    # Agency set to DEFTl, which only has 1 verifier
+    agencies = TransitAgency.all_active()[1:]
+    mocker.patch("benefits.core.models.TransitAgency.all_active", return_value=agencies)
+
+    assert len(agencies) == 1
+    assert len(agencies[0].eligibility_verifiers.all()) == 1
+
+    path = reverse("core:index")
+    response = client.get(path)
+
+    assert response.status_code == 302
+    assert response.url == reverse("eligibility:start")
+    # assert response.url == agencies[0].index_url
```

Comment:
@angela-tran Any ideas what is wrong with this spec? On lines 33/34, I'm checking that `agencies` is set to have 1 agency with 1 verifier -- so the data seems correct to me. The `response.url` is coming back as `/mst`, though, though I expect it to be going to `eligibility/start`.

Reply:
I believe what's happening is that when you call this code on lines 36 and 37,
```python
    path = reverse("core:index")
    response = client.get(path)
```
you're only going to get the first redirect. So it's only going through the first two pages:
```mermaid
graph LR
    A[core:index] -->|redirect| B(core:agency_index)
```

To make the pytest-django `client` fixture follow redirects, we can pass `follow=True` when calling `get`.

_From https://docs.djangoproject.com/en/3.2/topics/testing/tools/#making-requests:_
> If you set `follow` to `True` the client will follow any redirects and a `redirect_chain` attribute will be set in the response object containing tuples of the intermediate urls and status codes.

I tried it out locally, and here's what `redirect_chain` looks like:
![image](https://user-images.githubusercontent.com/25497886/173113503-a9981973-8c0f-4f24-afe3-daec6701a526.png)